main.js

```diff
@@ -1,22 +1,74 @@
 var $carContainer = document.querySelector('#car');
 var $car = document.querySelector('.car-size');
 var currentDirection;
+var intervalId = null;
+var $carContainerStyle = $carContainer.style;
+var topElf = 0;
+var leftShake = 0;
+var emptyInside = '';
+
 
 function movedirection(key) {
   if (key === 'ArrowUp') {
     $car.className = 'car-size rotate-up';
+    if(topElf === 0) {
+      return;
+    } else {
+      topElf -= 1;
+      emptyInside = 'top:'+topElf + 'px;left:'+leftShake+'px;position:absolute';
+      $carContainer.setAttribute('style',emptyInside);
+      clearInterval(intervalId);
+    }
   }
   if (key === 'ArrowLeft') {
     $car.className = 'car-size rotate-left';
+    if (leftShake === 0) {
+      return;
+    } else {
+      leftShake -= 1;
+      emptyInside = 'top:' + topElf + 'px;left:' + leftShake + 'px;position:absolute';
+      $carContainer.setAttribute('style', emptyInside);
```

Comment:
empty inside doesnt tell me anything about what the variable is for, please rename
and 
```
$carContainer.style.left
$carContainer.style.top
```
those exist ^